reddit_liveupdate/controllers.py

```diff
@@ -195,7 +195,11 @@ def _pixel_contents(self):
                 self._pixel_data = f.read()
         return self._pixel_data
 
-    def GET_pixel(self, event):
+    # this decorator takes **kwargs which routes treats specially and throws
```

Comment:
🔕 Haha, it took me a second to parse the words "routes treats specially", not realizing "routes" was a noun rather than a verb.